pkg/distro/rhel9/azure.go

```diff
@@ -456,6 +456,12 @@ var defaultAzureImageConfig = &distro.ImageConfig{
 				osbuild.NewModprobeConfigCmdBlacklist("amdgpu"),
 			},
 		},
+		{
+			Filename: "blacklist-intel-cstate.conf",
```

Comment:
Maybe a small comment in this codeblock that adds the blacklists that include the references from the commit message? (https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/deploying_rhel_9_on_microsoft_azure/assembly_deploying-a-rhel-image-as-a-virtual-machine-on-microsoft-azure_cloud-content-azure#making-configuration-changes_configure-the-image-azure) ? Or just somewhere in the file where we customize things (as it's more than just blacklisting that this describes)

Reply:
I added a comment at the beginning of the image config.